UI HOVER EFFECT/code/tilt.js

```diff
@@ -0,0 +1,183 @@
+var VanillaTilt = function () {
+    "use strict";
+    class t {
+        constructor(e, i = {}) {
+            if (!(e instanceof Node)) throw "Can't initialize VanillaTilt because " + e + " is not a Node.";
+            this.width = null, this.height = null, this.clientWidth = null, this.clientHeight = null, this.left = null, this.top = null, this.gammazero = null, this.betazero = null, this.lastgammazero = null, this.lastbetazero = null, this.transitionTimeout = null, this.updateCall = null, this.event = null, this.updateBind = this.update.bind(this), this.resetBind = this.reset.bind(this), this.element = e, this.settings = this.extendSettings(i), this.reverse = this.settings.reverse ? -1 : 1, this.glare = t.isSettingTrue(this.settings.glare), this.glarePrerender = t.isSettingTrue(this.settings["glare-prerender"]), this.fullPageListening = t.isSettingTrue(this.settings["full-page-listening"]), this.gyroscope = t.isSettingTrue(this.settings.gyroscope), this.gyroscopeSamples = this.settings.gyroscopeSamples, this.elementListener = this.getElementListener(), this.glare && this.prepareGlare(), this.fullPageListening && this.updateClientSize(), this.addEventListeners(), this.reset(), this.updateInitialPosition()
+        }
+        static isSettingTrue(t) {
+            return "" === t || !0 === t || 1 === t
+        }
+        getElementListener() {
```

Comment:
also make use of [private class methods and properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) if your method or property isn't going to be referenced from outside the class